src/Frontend/Core/Engine/Form.php

```diff
@@ -161,8 +161,8 @@ public function addDate(
             throw new Exception('A date field with type "range" should have 2 valid date-parameters.');
         }
 
-        // @later	get preferred mask & first day
-        $mask = 'd/m/Y';
+        // set mask and firstday
+        $mask = Model::getContainer()->get('fork.settings')->get('Core', 'date_format_short');
```

Comment:
You can just use `Model::get('fork.settings')->get('Core', 'date_format_short')`

Tip: if you commit this, you can use `git commit --amend` to add it to this commit. You'll need to force push though :)